src/poetry/installation/wheel_installer.py

```diff
@@ -25,6 +26,13 @@
 
     from poetry.utils.env import Env
 
+shebang = b"#!" + '/usr/bin/env python'.encode("utf-8")
```

Comment:
the string `#!/usr/bin/env python` is encoded the same in ascii and utf-8 - can you simplify?

Reply:
:+1: